packages/docsearch-react/package.json

```diff
@@ -36,7 +36,7 @@
   "dependencies": {
     "@algolia/autocomplete-core": "1.5.2",
     "@algolia/autocomplete-preset-algolia": "1.5.2",
```

Comment:
Why pin?

Reply:
_These maintainers could introduce breaking change anytime, I can feel it 👿_ 


We've always pinned the version here and rely on dependabot for the bump, wdyt?

Reply:
Not pinned is only useful if people install many versions of the package (indirectly).
In the wild it's "rarely" useful and I doubt people will have many autocomplete version in their node_modules.

I'm fine with it as long as we are regularly upgrading dependencies which I'm pushing for ☺️

Reply:
Not pinned can be nice with libraries when maintainers are "late" and the user wants to update transitive dependencies sooner. Even though we regularly upgrade dependencies, you might not release right away?

It seems like you're already not pinning `algoliasearch` so I thought it'd make sense. We do this in flavored versions of InstantSearch that rely on InstantSearch.js ([React](https://github.com/algolia/react-instantsearch/blob/master/packages/react-instantsearch-hooks/package.json#L52), [Vue](https://github.com/algolia/vue-instantsearch/blob/master/package.json#L49), [Angular](https://github.com/algolia/angular-instantsearch/blob/master/package.json#L95)).

Reply:
Ok that make sense! I don't think it's a blocker for this release PR but can definitely be revisited! Thanks for pointing it